### PR TITLE
submitted solution to week 4

### PR DESCRIPTION
**Pull request by Andrey8860**
An initial solution to week 4 module

### CODE REVIEW

**Review by mamax — COMMENTED** [review]
Put comments that need to be resolved. I would strongly suggest adding `.gitignore` file as well.

File: Week 4/Solution/src/main/java/com/epam/planes/MilitaryPlane.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public MilitaryType getType() {
+        return militaryPlaneType;
+    }
+
+    @Override
+    public String toString() {
+        return super.toString().replace("}", ", type=" + militaryPlaneType + '}');
+    }
+
+    @Override
```
[the remark this answers is not in the document]
Comment (Andrey8860) [reply Week 4/Solution/src/main/java/com/epam/planes/MilitaryPlane.java]: Resolved

File: Week 4/Solution/pom.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <version>1.0-SNAPSHOT</version>
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>7.0.0-beta1</version>
```
[the remark this answers is not in the document]
Comment (Andrey8860) [reply Week 4/Solution/pom.xml]: Resolved

File: Week 4/Solution/src/main/java/com/epam/planes/ExperimentalPlane.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+		super(planeModel, maximumPlaneSpeed, maximumPlaneFlightDistance, maximumPlaneLoadCapacity);
+		this.experimentalPlaneType = experimentalPlaneType;
+		this.classificationLevel = classificationLevel;
+	}
+
+	public ClassificationLevel getClassificationLevel() {
+		return classificationLevel;
+	}
+
+	public void setClassificationLevel(ClassificationLevel classificationLevel) {
```
[the remark this answers is not in the document]
Comment (Andrey8860) [reply Week 4/Solution/src/main/java/com/epam/planes/ExperimentalPlane.java]: Resolved

File: Week 4/Solution/src/main/java/com/epam/planes/ExperimentalPlane.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	public boolean equals(Object o) {
+		return super.equals(o);
+	}
+
+	@Override
+	public int hashCode() {
+		return super.hashCode();
+	}
+
+	@Override
```
[the remark this answers is not in the document]
Comment (Andrey8860) [reply Week 4/Solution/src/main/java/com/epam/planes/ExperimentalPlane.java]: Resolved

**Review by mamax — APPROVED** [review]

**Comment by mamax** [comment]
Looks good

**Pull request merged**